packages/integration-tests/projects/suite-web/tests/settings/t2-device-settings.test.ts

```diff
@@ -2,19 +2,24 @@
 // @retry=2
 
 describe('T2 - Device settings', () => {
+    beforeEach(() => {
+        cy.viewport(1080, 1440).resetDb();
+        cy.task('startBridge');
+    });
+    // cypress open todo: seems like entering urls (/settings/device) directly does not work anymore?
```

Comment:
It should and it works for me (e.g. https://suite.corp.sldev.cz/suite-web/develop/web/settings/device/)